app/views/search/index.html.haml

```diff
@@ -20,8 +20,15 @@
                 = t("search.advanced_options.ontologies")
               .field
                 - get_ontologies_data
-                = render Input::SelectComponent.new(id: 'search-ontologies', name: 'ontologies_list[]', value: @onts_for_select, multiple: "multiple", selected:  params[:ontologies_list])
-          
+                = render Input::SelectComponent.new(id: 'search_page_ontologies', name: 'ontologies_list[]', value: @onts_for_select, multiple: "multiple", selected:  params[:ontologies_list])
```

Comment:
this need to be a reusable helper, called `ontologies_selector_input` added in the file `ontologies_helper.rb`